src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -526,7 +513,6 @@ def _get_parent_proc_name():
     format_dict["cli_version"] = _get_az_version_summary()
     format_dict["python_info"] = "Python {}".format(platform.python_version())
     format_dict["platform"] = "{}".format(platform.platform())
-    format_dict["shell"] = "Shell: {}".format(_get_parent_proc_name())
```

Comment:
What's the consideration for removing psutil as we would lose shell information here?

Reply:
To bring back Cygwin support on Windows, psutil has compatibility issues. What's the purpose in collecting shell information? Why it cannot be provide by any other means?

Reply:
I agree with this trade. If future versions of psutil add support for Cygwin, we can then add it back.

Reply:
Thank you, I hope this change wouldn't cause to much impact for a feedback process flow

Reply:
@avoidik @fengzhou-msft, the reason we including shell info was because some issues are platform and / or  shell-based

cmd and powershell behave differently for example.....